### PR TITLE
DashboardScene - Add various-suite to scenes e2e

### LINKED ISSUE #88910: DashboardScene - add `various-suite` to scenes e2e

### PR DESCRIPTION
**Pull request by Sergej-Vlasov**
Fixes #88910

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by Sergej-Vlasov** [comment]
previous successful e2e test workflow run: https://github.com/grafana/grafana/actions/runs/9417424847/job/25942578181?pr=88911

**Comment by Sergej-Vlasov** [comment]
latest successful scenes e2e test run: https://github.com/grafana/grafana/actions/runs/9450627638/job/26030937521

**Review by ivanortegaalba — APPROVED** [review]
Go Go Go! 🔥

File: e2e/scenes/various-suite/frontend-sandbox-app.spec.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    describe('Sandbox disabled', () => {
+      beforeEach(() => {
+        cy.window().then((win) => {
+          win.localStorage.setItem('grafana.featureToggles', 'pluginsFrontendSandbox=0');
+        });
+      });
+
+      it('Loads the app page without the sandbox div wrapper', () => {
+        cy.visit(`/a/${APP_ID}`);
+        cy.wait(200); // wait to prevent false positives because cypress checks too fast
```
Comment (ivanortegaalba) [inline e2e/scenes/various-suite/frontend-sandbox-app.spec.ts:30]: I hate these `cy.wait` 😢

**Pull request merged**